styles/Editor.css

```diff
@@ -939,6 +939,57 @@
   font-size: 75%;
 }
 
+.ProseMirror .table-footer  {
+  font-size: 14px;
+  padding-top: 10px;
+}
+
+.ProseMirror .table-footer .footnote-element:not(:first-child)::before {
+  content: '';
+  border-bottom: 1px dashed #C9C9C9;
+  display: block;
+  margin-bottom: 8px;
+}
+
+.ProseMirror .table-footer .footnote-element {
+  counter-reset: table_footnote;
+}
+
+.ProseMirror .table-footer .footnote-element p::before {
+  counter-increment: table_footnote;
+  content: counter(table_footnote);
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+  vertical-align: super;
+  font-size: 75%;
+  margin-right: 4px;
+}
+
+.ProseMirror .table-footer p {
+  margin-bottom: 0px;
+  margin-top: 0px;
+  padding: 4px;
```

Comment:
Based on the designs, there is no such padding in the usual state (not selected).

Notice the first two items compared to the selected one:

![Screenshot 2024-01-22 at 11 54 41 AM](https://github.com/Atypon-OpenSource/manuscripts-body-editor/assets/33928809/f75f81df-16e5-465b-8dfb-22858d7452b6)

Reply:
adding padding to the selected state is leading to the jumpy animation which should be avoided by adding the padding to original p 

https://github.com/Atypon-OpenSource/manuscripts-body-editor/assets/93189633/746892ec-8c16-4406-be22-e8a0f3fa9219





and this is how it looks from my end 

![image](https://github.com/Atypon-OpenSource/manuscripts-body-editor/assets/93189633/6ccc721d-9cbf-4c7a-83c8-cc30390f7ae6)

Reply:
Yeah, I know. I also liked how you handled it but this doesn't look like the design. The screenshot I shared above is from the designs.

So we either accept jumping to align with the design paddings or violate the design.